deployment/quick-start/services-configuration.yaml.template

```diff
@@ -41,6 +41,10 @@ cluster:
     # Must be lower case, e.g., regsecret.
     secret-name: pai-secret
 
+  api-server:
```

Comment:
Service configuration? I don't think this section should be here.

Reply:
which position you think is better?